cmd/maya-apiserver/app/server/volume_endpoint_v1alpha1.go

```diff
@@ -185,11 +185,16 @@ func (v *volumeAPIOpsV1alpha1) read(volumeName string) (*v1alpha1.CASVolume, err
 
 	// use StorageClass name from header if present
 	scName := strings.TrimSpace(v.req.Header.Get(string(v1alpha1.StorageClassHeaderKey)))
+	patchVal := v.req.Header.Get(string(v1alpha1.NodeAffinityReplicaJivaIsPatchHeaderKey))
 	// add the StorageClass name to volume's labels
 	vol.Labels = map[string]string{
 		string(v1alpha1.StorageClassKey): scName,
 	}
 
+	vol.Annotations = map[string]string{
+		string(v1alpha1.NodeAffinityReplicaJivaIsPatchKey): patchVal,
```

Comment:
Can we use `IsPatchJivaReplicaNodeAffinity`?
```go
vol.Annotations = map[string]string{
  string(v1alpha1.CASKeyIsPatchJivaReplicaNodeAffinity): patchVal,
}
```

Reply:
This is annotations, which is of the form `nodeAffinity.replica.jiva.openebs.io/is-patch`. So above naming is done accordingly.